js/src/views/Account/account.js

```diff
@@ -74,10 +74,15 @@ class Account extends Component {
   componentWillReceiveProps (nextProps) {
     const prevAddress = this.props.params.address;
     const nextAddress = nextProps.params.address;
+    const { accounts } = nextProps;
 
     if (prevAddress !== nextAddress) {
       this.setVisibleAccounts(nextProps);
     }
+
+    if (!Object.keys(this.exportStore._accounts).length) {
```

Comment:
Yeah, still playing... had to sit on this one. Really the best idea is to wait to create the exportStore until componentWillReceiveProps.

Why not, theres no benefit to componentWillMount